package.json

```diff
@@ -50,8 +50,8 @@
     "@types/promise.allsettled": "^1.0.3",
     "@types/tsscmp": "^1.0.0",
     "axios": "^1.7.4",
-    "express": "^4.16.4",
-    "path-to-regexp": "^6.2.1",
+    "express": "^5.0.0",
+    "path-to-regexp": "^8.1.0",
```

Comment:
Just ugprading path-ot-regrexp does not resolve the npm-audit error yet. express v4.x still brings path-to-regexp v6